### DISCUSSION TITLE
How do slugs work?

### DISCUSSION THREAD

**Discussion by ShutdownRepo** [GitbookIO/community#275] [1. How to]
I have a space with multiple categories, one of which is "Active Directory", and the slug keeps reverting to "a-d" while I want it to be "ad". I change it every once in a while but GitBook seems to keep reverting it, and I can't find where the slug is set. I have my space synced with a GitHub repo and there is no occurrence of the "a-d" slug in this repo. Is there any way to stop GitBook changing it, or maybe understand better how it works?

**Comment by taszap** [comment]
Hey @ShutdownRepo thanks for this, it's interesting. 
I'm guessing this: https://www.thehacker.recipes/ad/movement is a space we are talking about? 

The slugs for pages are set by clicking on rename
<img width="359" alt="image" src="https://github.com/GitbookIO/community/assets/63643069/416ab28e-c1c7-4f2a-b9a1-ed2d63ccd7be">

This should allow you to change the slug to what you need it to be. 
<img width="356" alt="image" src="https://github.com/GitbookIO/community/assets/63643069/c19f7795-4bb4-403d-85b0-ca012eb3e5f8">

Do you have a specific CR or commit to use as an example when this was overwritten so we can take a deeper look?

**Reply by ShutdownRepo** [reply]
Yup, that's the space
And yes, that's where I edit the slug
I don't know exactly what make it change, hence my question, but basically, say I set the slug to "ad", sometimes I realize that the slug got re-set to "a-d", for a reason I don't know
Is the slug reflected somewhere in the .md structure ? /.gitbook/ ? Or something else
